README.md

```diff
@@ -81,88 +81,107 @@ When exporting, choose Template instead of HTML or Markdown. This can be combine
 Examples from [WaveDrom tutorials](http://wavedrom.com/tutorial.html).
 
 #### Simple Example
-```wavedrom
-{ signal : [
-  { name: "clk",  wave: "p......" },
-  { name: "bus",  wave: "x.34.5x",   data: "head body tail" },
-  { name: "wire", wave: "0.1..0." },
-  { name: "additional", wave: "0...1.." }
-]}
-```
+
```

Comment:
Can we remove the quote block (indented block) so that it shows up as a 'live demo' type of thing? Or maybe have both, code and live demo?

Reply:
@drjson yes, you are right. It was ment for the demo!
I will remove indent.